context/app/markdown/docs/Submitting_a_Dataset.md

```diff
@@ -0,0 +1,116 @@
+-   Imaging Mass Cytometry
+-   multiplexed IF
+-   PAS microscopy
+-   seqFISH
+-   MALDI-IMS positive
+-   MALDI-IMS negative
+
+   ### sequencing:
+-   bulk RNA
+-   bulk ATAC
+-   scRNA-Seq
+-   sci-ATAC-seq
+-   sci-RNA-seq
+-   SNARE-SEQ2 (RNAseq & ATACseq)
+-   snATAC
+-   snRNA
+-   SPLiT-Seq
+-   WGS
+
+### Assay Metadata
+Data centers provide the following 4 data types (Figure 3)for each data submission to HubMAP:
+**![Figure 3: Four datatypes in HuBMAP](https://lh4.googleusercontent.com/roCn5JFuGk3-tTn-n8wPL8cCOQ07t7vCZMyxuPI92LgDCIBFV4LPhKAIGgrL66b9XvuR45eeaAy9474jbfABdEoOVKam6hC0fBTshzNz0CMUaAOYhrfL3d3nsQN0VVbvV3KMMGVE)**
+
+Assay metadata, which is described in the [Assay Metadata Submission Format](https://docs.google.com/document/d/1g82GpCpFDKew60XzAO4Siaw3ZXJjwsaCpgPwhqQZxIY/edit#heading=h.qeehtnf68fas) document, is divided into 4 levels:
+
+### Definition of assay metadata levels
+
+-   Level 1: Are attributes that are common to all assays, for example, the type (“CODEX”) and category of assay (“imaging”), a timestamp, and the name of the person who executed the assay.
+    
+-   Level 2: Attributes that are common to a category of HuBMAP assays, i.e. imaging, sequencing, or mass spectrometry. For example, for imaging assays this includes fields such as x resolution and y resolution.
+    
+-   Level 3: Attributes that are specific to the type of assay, for example for CODEX that would include number of antibodies and number of cycles.
+    
+-   Level 4: Supplementary information such a QC report or information that is unique to a lab, not required for reproducibility or is otherwise not relevant for outside groups. This information is submitted in the form of a single file, a ZIP archive containing multiple files, or a directory of files. There is no formatting requirement (although formats readable with common tools such as text editors are preferable over proprietary binary formats).
+    
+#### Level 1 metadata is required for all assays:
+
+-- execution_datetime:  A time stamp marking the beginning of data capture for the assay. This typically comes from a date-time stamped folder generated by the acquisition instrument.
+-- protocols_io_doi: All HuBMAP protocols must be available through a protocols.io DOI link.
+-- operator:  The individual who conducted the assay.
+-- operator_email:
+-- pi: The principal investigator providing the data.
+-- pi_email:
+-- assay_category: One of 3 general assay categories- imaging, mass spectrometry or nucleotide sequencing.
+-- assay_type: The specific type of assay being executed (eg. CODEX, immunofluorescence microscopy, LC-mass spectrometry, RNAseq, ATACseq, etc)
+-- analyte_class:  Proteins, nucleic acids, metabolites, lipids.
+-- is_targeted:  A boolean indicating if the assay targets a specific analyte (eg. specific mRNA(s) or protein(s).
+
+  
+
+#### Level 2 metadata encompasses metadata relevant to an assay category (CODEX Level 2 metadata is shown here as an example):
```

Comment:
@cebriggs7135 Did you update the example with the latest metadata specification for CODEX? Otherwise this will be confusing. It might be better to just link to the corresponding metadata spec documentation in Github, which will always be up to date.

Reply:
The sample applies to levels 1 and 3 btw.

Reply:
I'll give the link to the CODEX assay page in Github where viewers can review all 3 levels of fields for CODEXhttps://github.com/hubmapconsortium/ingest-validation-tools/blob/master/docs/codex/README.md
![Screen Shot 2020-07-02 at 4 57 05 PM](https://user-images.githubusercontent.com/8284878/86408600-45721c80-bc85-11ea-899f-897348ad7455.png)